src/components/interfaces/MOBILE_API.xml

```diff
@@ -138,6 +138,9 @@
         <element name="CORRUPTED_DATA" since="5.0">
             <description>The data sent failed to pass CRC check in receiver end</description>
         </element>
+        <element name="ENCRYPTION_NEEDED" since="5.1">
```

Comment:
```suggestion
        <element name="ENCRYPTION_NEEDED" since="6.0">
```

Reply:
fixed in 45f0888b531adde8ac91715a3c526c75c7c23ba3